cmd/cclient/main.go

```diff
@@ -86,10 +89,28 @@ func send(recipient string, amountStr string) {
 	log.Printf("transaction %d cleared", transaction.Sequence)
 }
 
+func handler(w http.ResponseWriter, r *http.Request) {
+	pass :=strings.TrimLeft(r.URL.Path, "/")
```

Comment:
space after := is missing, maybe check that go fmt ran?

Reply:
I'll run it. Is it supposed to run automatically or something?